index.html

```diff
@@ -410,6 +463,94 @@ <h3 id="installation-sec">
       </h2>
       <section>
         <h3>
+          <code>BeforeInstallPromptEvent</code>
+        </h3>
+        <pre class="idl">
+enum AppBannerPromptOutcome {
+  "accepted",
+  "dismissed"
+};
+
+[Constructor(DOMString typeArg, optional BeforeInstallPromptEventInit eventInit)]
+interface BeforeInstallPromptEvent : Event {
+    readonly attribute Promise&lt;AppBannerPromptOutcome&gt; userChoice;
```

Comment:
Right now, yes. Still need to clarify what that is for? Let's discuss that in the bug.